src/typography/brand/Brand.story.mdx

```diff
@@ -0,0 +1,33 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { color } from '../../_utils/branding'
+import { TextBrand } from '.'
+
+<Meta title="Design Tokens/Typography" />
+
+# **TextBrand**
```

Comment:
Is it possible to copy SubHeader Story instead?
- Split stories
- Rename meta
- Use knobs